app/frontend/styles/application.scss

```diff
@@ -16,8 +16,9 @@ $govuk-image-url-function: frontend-image-url;
 @import 'base/mixins';
 @import 'base/utilities';
 @import 'components/awaiting_feedback_vacancies';
-@import '../src/lib/autocomplete';
+@import '../src/lib/ui/patterns/autocomplete';
 @import 'components/banners';
+@import 'components/accordian-group';
```

Comment:
accordian -> accordion

Reply:
This applies to code and to filenames

Reply:
just pushed to correct this everywhere is wrong